### ISSUE TITLE
Search result-related messaging missing status role

### ISSUE THREAD

**Issue by aviannamiller** [omeka/omeka-s#2402]
https://github.com/omeka/omeka-s/issues/2171 discusses the issue of missing status roles on admin-side error messages (the same being true for success messages). An additional area missing this kind of markup would be search result messaging (ex. "Omeka could not find any items", "There are no results", "Loading results...", "# results found"). This is true for **both** admin and public side searches. See [SC 4.1.3](https://w3.org/TR/WCAG22#status-messages) for more.
